client/scripts/index.js

```diff
@@ -108,4 +131,7 @@ socket.on('disconnect', reason => {
     window.location = '/'
 })
 
```

Comment:
can add a comment here explaining this is where you do all the socket handlers:
ex:
```
// action handlers after receiving messages from server:
socket.on('room_id', displayRoomID)
socket.on('new_image', update_image_from_server)
socket.on('player_number', disableButtons)
socket.on('enable_turn', enableButtons)
```